src/framework/standard/help_commands.rs

```diff
@@ -395,15 +395,17 @@ fn check_command_behaviour(
     let b = check_common_behaviour(&ctx, msg, &options, owners, help_options);
 
     if b == HelpBehaviour::Nothing {
-       for check in options.checks {
-           if !check.check_in_help {
-               break;
-           }
+       if !options.owner_privilege || !owners.contains(&msg.author.id) {
+           for check in options.checks {
+               if !check.check_in_help {
+                   break;
```

Comment:
This should also be fixed.
```suggestion
                   continue;
```
It's a bug to stop executing more checks if one check does not desire to be applied to the help command.

Reply:
Done.